plugins/zfs/zfs.go

```diff
@@ -32,7 +37,7 @@ func (z *Zfs) Description() string {
 	return "Read metrics of ZFS from arcstats, zfetchstats and vdev_cache_stats"
 }
 
-func getTags(kstatPath string) map[string]string {
```

Comment:
I was thinking about putting the directory scan into a getPools() that returns a slice of structs containing the pool name and io path. Then the getPoolsTag() and gatherPoolStats() will both take this this as the input. Do you like that better than adding a tags property to Zfs?